runtests.py

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env python3
 from os import system
-from sys import argv, exit, platform
+from sys import argv, exit, platform, executable, version_info
 
 prog, *args = argv
 
-if platform == 'win32':
-    python_name = 'py -3'
+
+# Use the Python provided to execute the script, or fall back to a sane default
+if version_info >= (3, 4, 0):
```

Comment:
As long as we're nit-picking here, I'd say `if version_info[0] >= 3` -- this speaks to our intention which is that we're really just distinguishing between (supported versions of) Python 2 and 3.

Reply:
Perhaps you missed my comment [here](https://github.com/python/mypy/pull/5702#issuecomment-426069096), but I think this is not the right thing to do. We want to select only a version of Python 3 that mypy supports.

Reply:
I did see that comment but wasn't sure how it mattered...

So the case where this matters is apparently one where runtests.py is explicitly invoked with an unsupported version of Python 3, e.g. if the user typed `python3.3 runtests.py` while the default `python3` refers to 3.4 or higher. Right? I can dig that, though it's too bad that we can't import mypy.defaults (and please don't try to find a way to be able to import that -- I think it's important that this script is clearly independent from the mypy package).

So now I agree.